src/schema/slack/functions/create_channel.ts

```diff
@@ -15,6 +15,12 @@ export default DefineFunction({
         description: "Enter a channel name",
         title: "Channel name",
       },
+      user_ids: {
+        type: SchemaTypes.array,
+        description: "Search all people",
```

Comment:
.. this seems like a weird field description. Just me or..?

Reply:
That does feel off putting, maybe it should be `search all users`, not sure who to ask about changing this

Reply:
After looking again, this seems consistent with our other built-ins. The title specifies what the field does, and the description tells the user what to do. I expect this field mostly is for WFB purposes

The same description is used in
- `AddUserToUsergroup`
- `InviteUserToChannel`
- `RemoveUserFromUsergroup`